.ci/docker/build.sh

```diff
@@ -460,10 +460,18 @@ if [[ "$image" == *cuda*  && ${OS} == "ubuntu" ]]; then
   fi
 fi
 
+no_cache_flag=""
+progress_flag=""
+# Do not use cache and progress=plain when in CI
+if [[ "${CI:-}" = "true" ]]; then
```

Comment:
I thought CI is just set to 1, rather than true. But don't we want to set those if this variable is defined to anything?
```suggestion
if [[ -n "${CI:-}"]]; then
```